### PR TITLE
alert components

### PR DESCRIPTION
**Pull request by mehulsatardekar**
alert components added.

Live Preview : https://framer-ui.netlify.app/docs/alert.html

### CODE REVIEW

**Review by rahulk31 — COMMENTED** [review]
Clean and elegant design. Code looks fine to me. Merge Ready PR 👍

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+/*alert link colors*/
+
+.alert-link{
+    color: var(--text-col-white);
+    font-weight: bold;
+}
+
+.alert-link:hover{
+    color: rgb(48, 47, 47);
+}
```
Comment (Sahil25061999) [inline components/alert/alert.css:69]: great ui, the overall code looks good, you can use variable over here.

**Comment by gauravsinhaweb** [comment]
Looks good to me. Nice work!

File: components/alert/alert.css (hunk 1 above)
Comment (mehulsatardekar) [reply components/alert/alert.css:69]: Thanks, @Sahil25061999  will do the changes.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/* @import url('../../components/root.css');
+@import url('../../components/Buttons/button.css'); */
+
+@import url('../../components/framer.css');
+
+:root{
+    --bg-icon-hover:#818cf8;
+}
+.alert{
```
Comment (veereshbv04) [inline components/alert/alert.css:9]: add close button to all alerts

File: docs/alert.html [hunk 3]
```diff
@@ -0,0 +1,399 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (veereshbv04) [inline docs/alert.html:2]: first page UI is awesome

**Comment by rahulyadav139** [comment]
The files are readable, you have put comments in the CSS which help me to what I am looking at.  The colour selection is great. Go ahead and merge the branch.

File: components/alert/alert.css (hunk 2 above)
Comment (mehulsatardekar) [reply components/alert/alert.css:9]: Yes, @veereshbv04. That's what the user will do the whole intention of my CL is it's highly customizable so even if they wish to add buttons, simply they can go to the buttons section of CL and import it on the alert section.

**Review by saurabhchirde — APPROVED** [review]
Loved the Landing page 🔥 Overall code is neat and clean, you can proceed to merge.

**Review by rahulyadav139 — APPROVED** [review]

**Review by rahulk31 — APPROVED** [review]

**Pull request merged**